web/carousel/src/components/UI/ConfirmPopup/ConfirmPopup.js

```diff
@@ -0,0 +1,29 @@
+import "react-confirm-alert/src/react-confirm-alert.css";
+import classes from "./ConfirmPopup.module.css";
+
+import { confirmAlert } from "react-confirm-alert";
+import ButtonPrimary from "../ButtonPrimary/ButtonPrimary";
+
+const handleSubmit = (title, handleConfirm) => {
+  confirmAlert({
+    customUI: ({ onClose }) => {
+      return (
+        <div className={classes.Popup}>
+          <h1>{title}</h1>
+          <div className={classes.ButtonRow}>
+            <ButtonPrimary
+              onClick={() => {
+                handleConfirm();
+                onClose();
+              }}
+              title="Yes"
+            />
+            <ButtonPrimary onClick={onClose} title="No" />
```

Comment:
We can use <ButtonSecondary> here, in order to reinforce the difference between "yes" and "no"

Reply:
Which one is better?
![image](https://user-images.githubusercontent.com/32341138/102384245-882e7380-3fdd-11eb-8818-a660f90c1209.png)
![image](https://user-images.githubusercontent.com/32341138/102384327-a5fbd880-3fdd-11eb-9a62-de1720298027.png)

Reply:
Hmm, I think i was imagining the SecondaryButton with an outline with empty background. Only text looks not that good.

Reply:
I think an outline is not necessary, but 'delete' and 'cancel' make the buttons more clear for users. I can change them.